src/Chapter1.hs

```diff
@@ -613,14 +624,17 @@ Implement a function that returns the sum of the last two digits of a number.
 6
 >>> sumLast2 134
 7
->>> sumLast 1
+>>> sumLast2 1
 1
 
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let lastDigit = mod (abs n) 10
+      preLastDigit = mod (div (abs n) 10) 10
```

Comment:
Great one!
You can see that this `last1` is actually the `lastDigit` function from the previous task.

Also, I notice that you use

```haskell
mod (abs n) 10
```
and 
```haskell
div (abs n) 10
```

One hint to make your solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(a, b) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Oh, I knew there is a function lastDigit (implemented by myself), but I thought local variable lastDigit in the scope of let expression would be picked by the compiler. I'll remember.
Nice suggestion, I'm going to rewrite the code to make use of divMod function.